package.json

```diff
@@ -24,9 +27,7 @@
     "redux-saga": "^0.15.3",
     "reselect": "^3.0.1",
     "webcomponentsjs": "^1.0.2",
-    "whatwg-fetch": "^1.0.0",
-    "@webcomponents/custom-elements": "^1.0.0",
-    "@webcomponents/webcomponentsjs": "^1.0.0"
+    "whatwg-fetch": "^1.0.0"
```

Comment:
@jgiardino, Are the whatwg-fetch, webcomponents/custom-elements, and webcomponents/webcomponentsjs are required?

Reply:
My understanding of whatwg-fetch is that it's used for fetching data. I'm assuming we are using it, although I could be wrong.

I added webcomponents/custom-elements and webcomponents/webcomponentsjs when I created the Tabs component. I'd like to replace these Tabs with the patternfly-react Tabs, but will do that in a later PR.